public/content/lessons/2017/derivative-formulas-geometrically/index.mdx

```diff
@@ -1,10 +1,381 @@
+
+<Figure
+  image="figures/power-rule-example.svg" 
+  caption="Derivative of $f(x) = 3x^{2}$ modeled as area"
+/>
+
+Looking at the figure we can see the rate of the change of the function represented by the new area of $6$ thin rectangles with area $x \cdot dx$. Alternatively, the question can be modeled as volume, where instead of scaling one of the sides, the $3$ extends the geometry into another dimension.
+
+<Figure
+  image="figures/3x^2-volume-model.svg" 
+  caption="Derivative of $f(x) = 3x^{2}$ modeled as volume"
+/>
+
+This gives us the equivalent derivative, as the new volume of the two rectangular prisms is equal to $3 \cdot x \cdot dx + 3 \cdot x \cdot dx = 6x$. As before, we are ignoring the smaller rectangular prism because its volume $3dx^2$ contains $dx^2$.
+
+</Question>
+
+### Derivative of $f(x) = \frac{1}{x}$
+
+As another example, think of the function $f(x) = \frac{1}{x}$. Now, on the one hand, you could blindly try applying the power rule, since $\frac{1}{x}$ is the same as writing $x^{-1}$. That would involve letting that $-1$ hop down, leaving behind one less than itself, $-2$. 
+But let’s have some fun and see if we can reasonable this geometrically, rather than just plugging it through a formula.
+
+The value $1/x$ is asking “what number multiplied by $x$ equals $1$”, so here’s how I’d visualize it: Imagine a little rectangular puddle of water in two dimensions with area $1$. Let’s say that it’s width is $x$, which means the other must be $1/x$, since the total area is $1$.
+
+<Figure
+  image="figures/10.51.svg" 
+/>
+
+For example, if you increase $x$ to $3$, the other side must be squished down to $\frac{1}{3}$. And if $x=2$, the other side is forced to be $\frac{1}{2}$. 
+
+<Figure
+  image="figures/10.58.svg" 
+/>
+
+This is a nice way to think about the graph of $1/x$, by the way. If you think of the width $x$ of this puddle in the $xy$-plane, the corresponding output $1/x$, the height of the graph above that point, is whatever height the puddle must have to maintain an area of $1$.
+
+<Figure
+  image="figures/11.00.svg"
+  video="figures/11.11-11.24-trimmed.mp4" 
+/>
+
+For the derivative, imagine nudging the input $x$ up by a value $dx$. How must the height of this rectangle change so that the area remains unchanged at $1$? That is, increasing the width by $dx$ adds some new area to the right here, so the puddle must decrease in height by some $d(1/x)$ so that the area lost off the top here cancels that out.
+
+<Figure
+  image="figures/11.41.svg" 
+/>
+
+You should think of that $d(1/x)$ as being some tiny *negative* value, since it’s *decreasing* the height of this rectangle. And you know what? I’m gonna leave the last few steps here for you to pause and ponder. And once you work out $d(1/x)/dx$, compare it to what happens if you apply the power rule purely symbolically to $x^{-1}$.
+
+$$
+\frac{d(1 / x)}{d x}= ? ? ?
```

Comment:
This is actually one we should explain.  Let's frame it as a multiple-choice question.

In the explanation, mention that the new area introduced by increasing the width is dx * (1 / x), and that the change in area caused by the reduction from the top is -x * d(1 / x).  Rearrange to find that d(1 / x) = (-1 / x^2) * dx.

Also mention that the little corner of area d(1 / x) * dx in the upper right has negligible influence as $dx \to 0$